CONTRIBUTING.txt

```diff
@@ -379,6 +379,27 @@ To rebuild a full clean documentation, run:
     make clean
     make html
 
+Gallery
```

Comment:
I think this should go under the requirements *and* I just saw that the requirements are missing sphinx-gallery!

Reply:
Makes sense. Moving this [sub * n]section as a [sub * (n + 1)]subsection under the Requirements [sub * n]section with bdfbbd245.